OVERVIEW.md

```diff
@@ -0,0 +1,148 @@
+# DSpace Command-Line Interface (CLI)
+## Overview
+
+This project serves to provide Classes and procedures implemented in JRuby for
+developing a command-line interface for performing operational tasks within a
+DSpace installation. However, it should be noted that, for those DSpace
+installations which are maintained by the Princeton University Library, there
+are additional tiers of functionality which are structured for usage from the
+command-line. Hence, the current state of DSpace CLI operations are structured
+as such:
+
+1. dspace-cli (JRuby)
+1. `tcsh` scripts (These typically wrap either the dspace-cli procedures, custom JRuby procedures, or the DSpace Java CLI)
+1. dspace-jruby (Underlying JRuby API)
+1. DSpace Java CLI (From DSpace core)
```

Comment:
Can we add the link? https://wiki.lyrasis.org/display/DSDOC5x/Command+Line+Operations
I assume this is what you went off?